addons/mail/static/src/models/thread/thread.js

```diff
@@ -452,6 +451,43 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * Performs the `execute_command` RPC on `mail.channel`.
+         *
+         * @static
+         * @param {Object} param0
+         * @param {integer} param0.channelId
+         * @param {string} param0.command
+         * @param {Object} [param0.postData={}]
+         */
+        static async performRpcExecuteCommand({ channelId, command, postData={} }) {
+            return this.env.services.rpc({
+                model: 'mail.channel',
+                method: 'execute_command',
+                args: [[channelId]],
+                kwargs: Object.assign({ command }, postData),
+            });
+        }
+
+        /**
+         * Performs the `execute_command` RPC on `mail.thread`.
```

Comment:
wrong method name in doc, and it's not on `mail.thread` (which does not exist as a model) but on given `threadModel`